proto/gk/v1/gatekeeper.proto

```diff
@@ -21,6 +23,14 @@ service Gatekeeper {
 		};
 	}
 
+	// Events
+	rpc ConsumeEvent(Event) returns (ConsumeEventResponse) {
+		option (google.api.http) = {
+			post : "/v1/events"
```

Comment:
did I stop being `events:pubsub` because Kafka would use the same endpoint?

Reply:
No, this isn't the endpoint for Pub/Sub messages but just an endpoint to consume events directly. My thinking is this would help with testing without having to convert events to Pub/Sub message data.